content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -163,36 +164,41 @@ public void init() {
 		// Populate similarity report headers used in generateSimilarityReport
 		SIMILARITY_REPORT_HEADERS.putAll(BASE_HEADERS);
 		SIMILARITY_REPORT_HEADERS.put(HEADER_CONTENT, CONTENT_TYPE_JSON);
+		
+		// Populate webhook generation headers used in setupWebhook
+		WEBHOOK_SETUP_HEADERS.putAll(BASE_HEADERS);
+		WEBHOOK_SETUP_HEADERS.put(HEADER_CONTENT, CONTENT_TYPE_JSON);
 
 		// Populate content upload headers used in uploadExternalContent
 		CONTENT_UPLOAD_HEADERS.putAll(BASE_HEADERS);
 		CONTENT_UPLOAD_HEADERS.put(HEADER_CONTENT, CONTENT_TYPE_BINARY);
 
 		try {
 			// Get the webhook url
-			webhookUrl = getWebhookUrl(Optional.empty());
+			String webhookUrl = getWebhookUrl(Optional.empty());
 			// Get list of existing webhooks, if any
 			ArrayList<Webhook> webhooks = getWebhooks();
 			
 			boolean webhooksSetup = false;
 			// Check to see if any webhooks exist for this url
 			for (Webhook webhook : webhooks) {
-				if (webhook.getUrl() == "http://vericite.com/content-review-tool/webhooks?providerName=TurnitinOC") {
+				if (StringUtils.isNotEmpty(webhook.getUrl()) && webhook.getUrl().equals(webhookUrl)) {
```

Comment:
you don't need an NPE check if you switch the equals around:

webhookUrl.equals(webhoooks.getUrl())